docs/sources/auth/jwt.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "JWT Authentication"
+description = "Grafana JWT Authentication Guide "
+keywords = ["grafana", "configuration", "documentation", "jwt"]
+aliases = ["/docs/grafana/latest/installation/jwt/"]
+weight = 300
++++
+
+# JWT Authentication
+
+You can configure Grafana to accept a JWT token provided in HTTP header.
+
+JWT token is being verified using a public key(s) stored in file or provided by JWKS endpoint.
+
+You can specify some of the json fields embedded in JWT (called claims) to use either as a login or as an email to sign in.
+
+```bash
```

Comment:
Please make these numbered lists, and update doc content to match the style in other topics.

Any example you want to show can be in code.